src/clj/test/cards/ice.clj

```diff
@@ -408,6 +408,43 @@
       (take-credits state :corp 2)
       (is (= 5 (:current-strength (refresh lotus))) "Lotus Field strength increased"))))
 
+(deftest mausolus
+  ;; Mausolus - 3 adv tokens change the subroutines
+  (do-game
+    (new-game (default-corp [(qty "Mausolus" 1)])
+              (default-runner[(qty "NetChip" 5)]))
+    (play-from-hand state :corp "Mausolus" "HQ")
+    (let [mau (get-ice state :hq 0)]
+      (core/rez state :corp mau)
+      (take-credits state :corp)
+      (run-on state :hq)
+      (is (= (:credit (get-corp)) 3) "corp starts encounter with 3 crs")
+      (is (= (count (:discard (get-runner))) 0) "runner starts encounter with no cards in heap")
+      (is (= (:tag (get-runner)) 0) "runner starts encounter with 0 tags")
```

Comment:
Love this! Stating assumptions etc. explicitly.

One thing some of us tend to do is to have the "constant" first in the comparison statement (so `(= 3 (:credit (get-corp)))`. I prefer it since it doesn't require you to count parentheses as much. Also makes the number align very prettily. Just personal preference though.

This could also apply to the conditionals in the `if` statements in the card definition, but once again, just personal preference :wink: